packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -112,123 +146,82 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
+                maxRounds,
+                skipRounds: RECOMMENDED_SKIP_ROUNDS,
+                targetAnonymity,
+            }),
+        );
 
     return (
         <>
             <StyledCard>
-                <Heading>
-                    <Translation id="TR_START_ANONYMISATION" />
-                </Heading>
-                <Row>
-                    <StyledCryptoAmountWithHeader
-                        header={
-                            <AmountHeading>
-                                <Translation id="TR_AMOUNT" />
-                            </AmountHeading>
-                        }
-                        value={notAnonymized}
-                        symbol={account.symbol}
-                    />
-                    <StyledCryptoAmountWithHeader
-                        header={
-                            <AmountHeading>
-                                <Translation id="TR_SERVICE_FEE" />
-                            </AmountHeading>
-                        }
-                        value={serviceFee}
-                        symbol={account.symbol}
-                        note={<Translation id="TR_SERVICE_FEE_NOTE" />}
-                    />
-                </Row>
-                {isCustom ? (
-                    <CoinjoinCustomStrategy
-                        maxRounds={maxRounds}
-                        maxFee={customMaxFee}
-                        skipRounds={customSkipRounds}
-                        setMaxFee={setCustomMaxFee}
-                        setSkipRounds={setCustomSkipRounds}
-                        reset={reset}
-                    />
-                ) : (
-                    <CoinjoinDefaultStrategy
-                        maxRounds={maxRounds}
-                        feeRatesMedians={coordinatorData.feeRatesMedians}
-                        strategy={strategy}
-                        setStrategy={setStrategy}
-                    />
-                )}
-            </StyledCard>
-
-            <StyledCard>
-                <ConfirmationsHeading>
-                    <Translation id="TR_CONFIRMATIONS" />
-                </ConfirmationsHeading>
-                <CheckboxWrapper>
-                    <StyledCheckbox
-                        isChecked={connectedConfirmed}
-                        onClick={toggleConnectConfirmation}
-                        data-test="@coinjoin/checkbox-1"
-                    >
-                        <Translation id="TR_DEVICE_CONNECTED_CONFIRMATION" />
-                    </StyledCheckbox>
-                    <TooltipSymbol content={<Translation id="TR_DEVICE_CONNECTED_TOOLTIP" />} />
-                </CheckboxWrapper>
+                <TopRow>
+                    <Translation id="TR_COINJOIN_SETUP" />
+                </TopRow>
+                <Tiles>
+                    {tiles.map(tile => (
+                        <Tile
+                            key={tile.description.props.title}
```

Comment:
isn't that undefined?
<img width="522" alt="image" src="https://user-images.githubusercontent.com/3729633/214810652-46220385-1887-4fce-81c7-f99274ed36d5.png">

Reply:
5eca8e7fc665902d1c5124883fa1cefb2034f08f